tutor/src/components/task-step/step-with-reading-content.cjsx

```diff
@@ -43,6 +44,7 @@ ReadingStepContent = React.createClass
           html={content_html}
         />
       </div>
+      {if stepType == 'reading' then <AnnotationWidget documentId={this.getCnxId()} />}
     </div>
```

Comment:
A typical React way to do things like this is to make a `AnnotationWidgetWrapper` in charge of deciding whether it should render.

something like:

```javascript
export default function AnnotationWidgetWrapper({ contentType , documentId }) {
  if ( -1 === ['reading', 'reference-book'].indexOf(contentType){ return null; }
 return <AnnotatorWidget documentId={documentId} />;
}
```

Then just render the wrapper here:
```javascript
<AnnotationWidgetWrapper contentType={stepType} documentId={this.getCnxId()} />
```